src/analysis/retail/mage/fire/guide/SunKingsBlessing.tsx

```diff
@@ -0,0 +1,167 @@
+      this.sunKingsBlessing.hotStreaksWithSKBThresholds.isGreaterThan.average;
+    const majorThreshold = this.sunKingsBlessing.hotStreaksWithSKBThresholds.isGreaterThan.major;
+    let performance = QualitativePerformance.Fail;
+    if (resets <= minorThreshold) {
+      performance = QualitativePerformance.Perfect;
+    } else if (resets <= averageThreshold) {
+      performance = QualitativePerformance.Good;
+    } else if (resets <= majorThreshold) {
+      performance = QualitativePerformance.Ok;
+    }
+    return performance;
+  }
+
+  get guideSubsection(): JSX.Element {
+    const combustion = <SpellLink spell={TALENTS.COMBUSTION_TALENT} />;
+    const pyroblast = <SpellLink spell={TALENTS.PYROBLAST_TALENT} />;
+    const fireBlast = <SpellLink spell={SPELLS.FIRE_BLAST} />;
+    const phoenixFlames = <SpellLink spell={TALENTS.PHOENIX_FLAMES_TALENT} />;
+    const sunKingsBlessing = <SpellLink spell={TALENTS.SUN_KINGS_BLESSING_TALENT} />;
+
+    const sunKingsBlessingIcon = <SpellIcon spell={TALENTS.SUN_KINGS_BLESSING_TALENT} />;
+    const hotStreakIcon = <SpellIcon spell={SPELLS.HOT_STREAK} />;
+    const combustionIcon = <SpellIcon spell={TALENTS.COMBUSTION_TALENT} />;
+
+    const explanation = (
+      <>
+        <div>
+          Because <b>{sunKingsBlessing}</b> gives you {combustion}, you typically treat this talent
+          similarly to {combustion} with a few changes/exceptions. Additionally, {sunKingsBlessing}{' '}
+          can be used to extend an existing {combustion} to make it last longer.
+        </div>
+        <div>
+          Follow the same rules and guidelines as {combustion} with the below adjustments and
+          additions:
+          <ul>
+            <li>Do not let {sunKingsBlessing} expire (if possible)</li>
+            <li>
+              Do not delay activating your {sunKingsBlessing} (even if {combustion} is coming off
+              cooldown soon).
+            </li>
+            <li>
+              You should still pool {fireBlast} and {phoenixFlames} for {sunKingsBlessing}, but you
+              will not need as many charges since the {combustion} buff will be shorter.
+            </li>
+            <li>
+              If you are activating {sunKingsBlessing} and {combustion} at the same time, make sure{' '}
+              {combustion} is activated BEFORE the hardcast {pyroblast} for {sunKingsBlessing}{' '}
+              finishes. If {combustion} is cast first, then {sunKingsBlessing} will extend the
+              duration of {combustion} (12s + 6s), but if {combustion} is activated after{' '}
+              {sunKingsBlessing}, then the buff duration will be reset to 12s.
+            </li>
+          </ul>
+        </div>
+      </>
+    );
+    const expiredTooltip = (
+      <>{this.sunKingsBlessing.sunKingExpireThresholds.actual} Expired Procs.</>
+    );
+    const wastedHotStreakTooltip = (
+      <>{this.sunKingsBlessing.hotStreaksWithSKBThresholds.actual} Wasted Hot Streaks</>
```

Comment:
This should prolly be rounded
```suggestion
      <>{this.sunKingsBlessing.hotStreaksWithSKBThresholds.actual.toFixed(2)} Wasted Hot Streaks</>
```
![image](https://github.com/WoWAnalyzer/WoWAnalyzer/assets/3404958/2edcaf3d-8f4a-4905-9347-16c828cbd6e1)

Reply:
Oops, thought I caught all of those but I Guess I missed that one